main.py

```diff
@@ -24,22 +29,26 @@ def click_button():
 def get_agency_list():
     container_element = browser_lib.find_element('id:agency-tiles-container')
     container_text = container_element.text.split('\n')
+
     result_list = list(zip(container_text[::4], container_text[2::4]))
     result_list.insert(0, ("Agency name", "Spending"))
+
     return result_list
 
 
-def write_to_excel(sheetname, bookname, content):
+def write_to_excel(sheetname, bookname, content, method):
```

Comment:
why did you add the **method** argument? The previous solution is better and more optimal.

Reply:
The previous solution was only appending data to excel file. It means that for each new launching of the program, excel files contains data from previous launch and gets appended with new data. I add method option to make usage of function more flexible. Now function is using to create book for agencies list (method "new") and to add sheet with spendings to it (method "append")

Reply:
Checkout to the previous function solution